File: src/UnicaffeServidor/src/main/java/br/edu/unilab/unicaffe/model/Laboratorio.java
package br.edu.unilab.unicaffe.model;

/**
 * Esta classe representa um local de acesso de computadores, provido de máquinas com UniCaffé. 
 * 
 * @author Jefferson Uchôa Ponte
 *
 */
public class Laboratorio {

	/**
	 * Id do laboratório no banco de dados. 
	 */
	private int id;
	/**
	 * Nome do laboratório. 
	 */
	private String nome;
	/**
	 * Perfil deste laboratório. 
	 */
	private Perfil perfil;
	/**
	 * @return id do laboratório
	 */
	public Laboratorio() {
		this.perfil = new Perfil();
	}
	public Perfil getPerfil() {
		return perfil;
	}
	public void setPerfil(Perfil perfil) {
		this.perfil = perfil;
	}
	public int getId() {
		return id;
	}
	/**
	 * @param id é o ID do laboratório no banco de dados. 
	 */
	public void setId(int id) {
		this.id = id;
	}
	/**
	 * 
	 * @return nome do laboratório 
	 */
	public String getNome() {
		return nome;
	}
	/**
	 * @param nome nome do laboratório. 
	 */
	public void setNome(String nome) {
		this.nome = nome;
	}


}

File: classes/model/Laboratorio.php
<?php

/**
 * Esta classe representa um local de acesso de computadores, provido de máquinas com UniCaffé.
 *
 * @author Jefferson Uchôa Ponte
 *
 */
class Laboratorio
{

    private $id;

    private $nome;
    
    private $perfil;
    
    public function setPerfil(Perfil $perfil){
        $this->perfil = $perfil;
    }
    
    public function getPerfil(){
        return $this->perfil;
    }

    public function __construct(){
        $this->id = 0;
        $this->perfil = new Perfil();
        
    }
    /**
     *
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     *
     * @return number
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     *
     * @param string $nome
     */
    public function setNome($nome)
    {
        $this->nome = $nome;
    }

    /**
     *
     * @return string
     */
    public function getNome()
    {
        return $this->nome;
    }
}

?>